package.json

```diff
@@ -24,6 +24,16 @@
     "email": "joel@oblador.se"
   },
   "homepage": "https://github.com/oblador/react-native-store-review",
+  "devDependencies": {
+    "@types/react": "^16.9.19",
```

Comment:
You didn't add any types did you? Would be cool if you did, but if not then this dependency is redundant.

Reply:
it is already a  index.d.ts-file available and that was working without this I suppose so I have removed it.

Reply:
you ment using types in this project of course, silly me 🤦